### PR TITLE
[MINOR] Add roadmap for 0.13 plus links to JIRA release notes

### PR DESCRIPTION
**Pull request by gweidner**
Updated roadmap for release 0.13 and added release notes.  Also added links to JIRA release notes for each prior release.

### CODE REVIEW

**Comment by gweidner** [comment]
Marking as [WIP] until RC2 approved by IPMC.

**Comment by gweidner** [comment]
Vote has been approved by IPMC:
https://www.mail-archive.com/general@incubator.apache.org/msg58834.html

File: 0.13.0-incubating/release_notes.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: SystemML 0.12.0-incubating Release Notes
```
Comment (deroneriksson) [inline 0.13.0-incubating/release_notes.md]: Please update line 3 to 0.13.0

**Comment by deroneriksson** [comment]
LGTM aside from minor title update. Thank you @gweidner! Please merge when appropriate.

**Comment by gweidner** [comment]
Thank you @deroneriksson for catching that!

**Comment by asurve** [comment]
We have 3 votes and voting ended on March 2nd.

**Comment by gweidner** [comment]
I've rebased and pushed new commit that corrects the markdown title, approval date, and supported spark version in download page and will merge to git.

**Pull request closed without merging**